store/seed/release/README.md

```diff
@@ -1,4 +1,3 @@
-This folder contains the seed data for release. The release seed data is used when building with "release" tag or running with --demo.
+This is used for local development when building without "release" tag (demo.bytebase.com and staging.bytebase.com).
```

Comment:
This is under release folder, so the wording is a bit confusing.

Reply:
Done.